lib/src/screens/recipe_screen.dart

```diff
@@ -159,14 +153,45 @@ class _RecipeScreenBodyState extends State<RecipeScreenBody> {
 
     final bottom = SliverList(
       delegate: SliverChildListDelegate.fixed([
-        if (recipe.tool.isNotEmpty) ToolList(recipe: recipe),
         if (MediaQuery.of(context).size.width > 600)
-          Row(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: list.map((e) => Expanded(flex: 5, child: e)).toList(),
-          )
+          ...[
+            Row(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                if (recipe.tool.isNotEmpty)
+                  Expanded(
+                    flex: 5,
+                    child: ToolList(recipe: recipe),
+                  ),
+                if (recipe.nutritionList.isNotEmpty)
+                  Expanded(
+                    flex: 5,
+                    child: NutritionList(recipe.nutritionList),
+                  ),
```

Comment:
Missing required trailing comma.

Please make sure your tests are passing static analysis my running `dart analyze` in the repo

Reply:
Thanks for catching that. I added in the commas and it is passing `dart analyze` now.